extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -118,10 +118,13 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 		computeInfoAndLinks.addItem(infoComputeStorage_p6, { CSSStyles: { 'margin-right': '5px' } });
 		content.addItem(computeInfoAndLinks, { CSSStyles: { 'min-height': '30px' } });
 
-
+		content.addItem(this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.workerNodes,
+			CSSStyles: { ...cssStyles.title, 'margin-top': '25px' }
+		}).component());
 
 		this.workerContainer = this.modelView.modelBuilder.divContainer().component();
-		this.handleServiceUpdated();
+		this.workerContainer.addItems(this.createUserInputSection(), { CSSStyles: { 'min-height': '30px' } });
```

Comment:
nit: could chain the items to the return value of divContainer() above using withItems call as in:
this.workerContainer = this.modelView.modelBuilder.divContainer()
     .withItems(this.createUserInputSection(), { CSSStyles: { 'min-height': '30px' } })
     .component();